.github/workflows/backend-api-push-to-main.yml

```diff
@@ -69,7 +70,7 @@ jobs:
 
     - name: Build test image
       run: |
-        docker build -t $BACKEND_API_DEV_TEST_IMAGE_REPOSITORY:$IMAGE_TAG . 
+        docker build -t $BACKEND_API_DEV_TEST_IMAGE_REPOSITORY:$IMAGE_TAG -t $BACKEND_API_BUILD_TEST_IMAGE_REPOSITORY:$IMAGE_TAG . 
```

Comment:
Are there any implications of us pushing a Docker Image to Build with a Dev tag and vice versa?

Reply:
I think it's only adding the `latest` tag here (?)

Reply:
Sorry I should have been clearer in my comment it relates more to the name of the image `$BACKEND_API_DEV_TEST_IMAGE_REPOSITORY` and `$BACKEND_API_BUILD_TEST_IMAGE_REPOSITORY` as opposed to `latest`.

I can't see anything in ECR that would but just curious